sklearn/utils/testing.py

```diff
@@ -697,5 +705,27 @@ def check_skip_travis():
     if os.environ.get('TRAVIS') == "true":
         raise SkipTest("This test needs to be skipped on Travis")
 
+class TempMemMap(object):
+    def __init__(self, X):
+        self.temp_folder = tempfile.mkdtemp()
```

Comment:
Maybe we could use a `prefix=sklearn_testing_` prefix  to make it explicit that this folder has been created by scikit-learn tests.